src/robusta/core/reporting/blocks.py

```diff
@@ -434,3 +436,41 @@ def grade(self):
             return "E"
         else:
             return "F"
+
+
+class EventsBlock(TableBlock):
+    """
+    Table display of a Events, Persists the events on the Robusta Platform.
+
+    Note: Wider tables appears as a file attachment on Slack, because they aren't rendered properly inline
+    """
```

Comment:
@pablos44 , this is the issue for the blocks classes.
Pydantic does not support positional arguments, only named ones.  seems like v2 might support it in the future.

So for every constructor we have ( around 14 ) we need to add the param name in the constructor.
```
    alert.add_enrichment(
        [
            TableBlock(
                <missing> rows=[[k, v] for (k, v) in labels.items()],
                <missing> headers=["label", "value"],
                table_name="*Alert labels*",
            ),
        ],
        annotations={SlackAnnotations.ATTACHMENT: True},
    )
```

Some extra info.
https://github.com/pydantic/pydantic/issues/116

I would prefer to create a new PR after this one with the changes on all constructor calls.